module2/assignment2.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Keating | INFO 2439 | Assignment #2</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="_css/keating_a2.css">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+</head>
+<body>
+<!-- Collin Keating
+     assignment2.html
+     INFO2439
+     Thoendel
+     9/14/19
+-->
+<div data-role="page" id="homePage" data-title="Keating | INFO 2439 | Assignment #2" data-add-back-btn="true" data-back-btn-text="Back">
+<div data-role="header" data-position="fixed"><a data-transition="slide" href="#" data-direction="reverse" class="ui-btn-left ui-btn ui-btn-inline ui-mini ui-corner-all ui-btn-icon-right ui-icon-back">Back</a></div>
+<div role="main" class="ui-content">
+            <h2>Mobile Application Research</h2>
+            <ul data-role="listview" data-inset="true">
+                <li><a data-transition="slide" href="#rmobileApp1">Jimmy Johns</a></li>
+                <li><a data-transition="slide" href="#rmobileApp2">Baskin Robbins</a></li>
+                <li><a data-transition="slide" href="#rmobileApp3">Dominos</a></li>
+            </ul>
+        </div>
+		<div data-role="footer" data-position="fixed"></div>
+		</div>
+		 <div data-role="page" id="rmobileApp1" data-title="Jimmy John's">
```

Comment:
I'd like to see this div flush left since we are starting a new page. Best practice indentation makes code much easier to maintain. I'd also like to see some text in your headers or  a heading as we've done on past pages. That lack of text or placeholder causes that awkward Back button on top of content. There was a section in your book (p. 30) that uses a <span> tag in these situations.